examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,536 @@
+show_anatomical_slices(tractogram_img.get_fdata(), 'DTI Density Map')
+
+print("Segmenting fiber groups...")
+
+segmentation = seg.Segmentation(return_idx=True,
+                                filter_by_endpoints=False)
+
+segmentation.segment(bundles,
+                     tractogram,
+                     fdata=hardi_fdata,
+                     fbval=hardi_fbval,
+                     fbvec=hardi_fbvec,
+                     mapping=mapping,
+                     reg_template=MNI_T2_img)
+
+fiber_groups = segmentation.fiber_groups
+
+for bundle in bundles:
+    tractogram = StatefulTractogram(fiber_groups[bundle]['sl'].streamlines,
+                                    img,
+                                    Space.VOX)
+    tractogram.to_rasmm()
+    save_tractogram(tractogram, op.join(working_dir, f'afq_{bundle}_seg.trk'),
+                    bbox_valid_check=False)
+
+    tractogram_img = density_map(tractogram, n_sls=1000, to_vox=True)
+    nib.save(tractogram_img, op.join(working_dir,
+                                     f'afq_{bundle}_seg_density_map.nii.gz'))
+    show_anatomical_slices(tractogram_img.get_fdata(),
+                           f'Segmented {bundle} Density Map')
+
+##########################################################################
+# Cleaning:
+# ---------
+# Each fiber group is cleaned to exclude streamlines that are outliers in terms
+# of their trajectory and/or length.
+#
+# .. note::
+#   By default cleaning
+#
+#   - resamples streamlines to 100 points
+#
+#   - given there are more than 20 streamlines cleaining will make maximum 5
+#     attempts to prune streamlines that are:
+#
+#      - greater than 5 standard deviations from the mean Mahalanobis distance,
+#        or
+#
+#      - greather than 4 standard deviations from the mean length
+#
+
+
+print("Cleaning fiber groups...")
+for bundle in bundles:
+    print(f"Cleaning {bundle}...")
+    print(f"Before cleaning: {len(fiber_groups[bundle]['sl'])} streamlines")
+    new_fibers, idx_in_bundle = seg.clean_bundle(
+        fiber_groups[bundle]['sl'],
+        return_idx=True)
+    print(f"After cleaning: {len(new_fibers)} streamlines")
+
+    idx_in_global = fiber_groups[bundle]['idx'][idx_in_bundle]
+    np.save(op.join(working_dir, f'{bundle}_idx.npy'), idx_in_global)
+
+    tractogram = StatefulTractogram(new_fibers.streamlines,
+                                    img,
+                                    Space.VOX)
+    tractogram.to_rasmm()
+    save_tractogram(tractogram, op.join(working_dir, f'afq_{bundle}.trk'),
+                    bbox_valid_check=False)
+
+    tractogram_img = density_map(tractogram, n_sls=1000, to_vox=True)
+    nib.save(tractogram_img, op.join(working_dir,
+                                     f'afq_{bundle}_density_map.nii.gz'))
+    show_anatomical_slices(tractogram_img.get_fdata(),
+                           f'Cleaned {bundle} Density Map')
+
```

Comment:
Maybe add an html/plotly visualization with the resulting segmented bundle?

Reply:
Use AFQ.viz.plotly_backend function for showing streamlines. This should generate an HTML file that you can then incorporate here using: 

    plotly.io.show(html_file_name)

Reply:
Addressed, but depends on #513

Reply:
Looks like that's been merged. Could you please rebase on top of master one more time?

Reply:
Oh - nevermind. Looks like it's working. I'll go ahead and merge.